### ISSUE TITLE
Runtime Hang observed in DETR-Segmentation variant

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-xla#4277]
### Describe the bug 

- Runtime Hang observed in `detr/segmentation/pytorch-ResNet50_Backbone_Panoptic`
- Hang originates from this [softmax](https://github.com/huggingface/transformers/blob/52d24b018e7aa12264b68ae2e162f9aadb26005b/src/transformers/models/detr/modeling_detr.py#L873) op

### Repro

```
git checkout kkannan/apr16_detr_seg_softmax_hang
git submodule update --init --recursive

pytest tests/runner/test_models.py::test_all_models_torch[detr/segmentation/pytorch-ResNet50_Backbone_Panoptic-single_device-inference] -svv

pytest tests/torch/ops/test_softmax.py -svv
```

### Logs

- [apr16_detr_seg.log](https://github.com/user-attachments/files/26784963/apr16_detr_seg.log)
- [apr16_softmax.log](https://github.com/user-attachments/files/26785013/apr16_softmax.log)
- [apr16_softmax_emitpy_gen.log](https://github.com/user-attachments/files/26785025/apr16_softmax_emitpy_gen.log)
- [apr16_softmax_emitpy_run.log](https://github.com/user-attachments/files/26785026/apr16_softmax_emitpy_run.log)
- [softmax.zip](https://github.com/user-attachments/files/26785027/softmax.zip)

**Comment by kamalrajkannan78** [comment]
tt-metal fix tracked by tt-xla now.
